docs/conventions.md

```diff
@@ -89,6 +89,16 @@ No conventions yet
 - Contract __MUST__ exist only in case existing logic provides for an extension by the user.
 - Contract is a public API and __MUST__ follow [Spryker plugin interfaces specification](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1038092073/INTEGRATED+RFC+Plugin+interface+specification).
 
+## REST API
+- Controller __SHOULD__ have no business logic.
+- Controller __SHOULD__ be named by template of task's id.
+  For task `validation:php:codestyle` - controller name is `ValidationController` and controller's action name is `validationPhpCodestyle` (task's full id in camelCase).
+- Controller __MUST__ be placed in namespace `SprykerSdk\Sdk\Presentation\RestApi\Controller\v1`. The `v1` is current version of SDK API.
+- Route __MUST__ be placed in `src/Presentation/RestApi/Resources/routing.yaml`
+- Route __SHOULD__ be named by template `api_${version}_${controller_name}_${action_name}`.
```

Comment:
This is how the route automatically generates by Symfony. Please, change this statement according to my comment above.

Reply:
Fixed